lib/pbench/test/unit/server/query_apis/test_index_mappings.py

```diff
@@ -30,7 +30,7 @@ def test_query(self, client, server_config, find_template):
         response body.
         """
         with client:
-            response = client.get(f"{server_config.rest_uri}/index/mappings")
+            response = client.get(f"{server_config.rest_uri}/index/mappings/run")
```

Comment:
I know, only the run `Template` is mocked, but ... it'd be good to verify that we can access a different template here. You can either duplicate the mock with a different name, or have the fixture create multiple template mocks and return them from a dict by name.